styles/styles.css

```diff
@@ -39,6 +39,211 @@ body {
     line-height: 0.8;
 }
 
+.topsection__container {
+    background-image: url("./../images/topsection_background.jpg") ;
+    background-size: cover;
+    background-attachment: scroll;
+    background-repeat: no-repeat;
+    background-position: top;
+    max-width: 1440px;
+    min-height: 100vh;
+    padding: 0;
+}
+
+.topsection__header {
+    display: flex;
+    justify-content: space-between;
+    padding: 51px 0 202px;
+    max-width: 1232px;
+    margin: 0 auto;
+}
+
+.topsection__logo {
+    background-image: url("../images/topsection_sprite.png");
+    background-repeat: no-repeat;
+    background-position-x: 0;
+    width: 205px;
+}
+
+.topsection__header__items {
+    display: flex;
+    gap: 51px;
+}
+
+.topsection__header__links {
+    list-style: none;
+    display: flex;
+    gap: 24px;
+    justify-content: space-around;
+    align-items: center;
+    color: #ffffff;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 24px;
+    line-height: 1.17;
+}
+
+.topsection__header__link {
+    position: relative;
+}
+
+.active::after {
+    position: absolute;
+    display: block;
+    content: "";
+    width: 90%;
+    height: 4px;
+    background-color: #f5bd41;
+    left: 4px;
+    bottom: -10px;
+}
+
+.topsection__header__buttons {
+    max-width: 94px;
+    display: flex;
+    justify-content: space-between;
+    gap: 24px;
+}
+
+.topsection__button__night {
+    height: 40px;
+    width: 30px;
+    background-image: url("../images/topsection_sprite.png");
+    background-position-x: -205px;
+}
+
+.topsection__button__account {
+    height: 40px;
+    width: 40px;
+    background-image: url("../images/topsection_sprite.png");
+    background-position-x: -235px;
+}
+
+.topsection__title {
+    color: #ffffff;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 50px;
+    line-height: 1.18;
+    text-align: left;
+    padding-left: 104px;
+    margin-bottom: 145px;
+}
+
+.topsection__form {
+    display: flex;
+    background: #FFFFFF;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 1.16;
+    height: 64px;
+    border-radius: 8px;
+    margin: 0 104px 120px;
+}
+
+.input {
+    color: #383838;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 1.17;
+    border-radius: 8px;
+    border: 3px solid transparent;
+    width: 100%;
+    height: 100%;
+}
+
+.topsection__city {
+    position: relative;
+    flex: auto;
+    padding-left: 22px;
```

Comment:
Move this padding into the input if you want it to be highlighted properly if active.